include/usearch/index_punned_dense.hpp

```diff
@@ -609,12 +638,14 @@ class index_punned_dense_gt {
         auto typed_result = typed_->copy(config);
         if (!typed_result)
             return result.failed(std::move(typed_result.error));
+#if USEARCH_LOOKUP_LABEL
         if (!result.index.free_ids_.reserve(free_ids_.size()))
             return result.failed(std::move(typed_result.error));
         for (std::size_t i = 0; i != free_ids_.size(); ++i)
             result.index.free_ids_.push(free_ids_[i]);
 
         result.index.labeled_lookup_ = labeled_lookup_;
```

Comment:
where would copy be used? 
Can the old index be used while/after copying? 
if so, should this not keep track of the old lookup table lock as well?